src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -289,14 +296,13 @@ RadixContainer<T> materialize_input(const std::shared_ptr<const Table>& in_table
   // Create histograms per chunk
   histograms.resize(chunk_count);
 
-  auto jobs = std::vector<std::shared_ptr<AbstractTask>>{};
+  std::vector<std::shared_ptr<AbstractTask>> jobs;
```

Comment:
Why?

Reply:
Probably readability, it is shorter and not slower here. However, the Hyrise style guide prefers **auto to stick** (https://www.fluentcpp.com/2018/09/28/auto-stick-changing-style/) and the drawbacks of auto are not too bad here, so maybe just go with it for the sake of consistency. It's not wrong, it's just convention. BTW, in Skyrise we opted the other way to be more consistent with the popular Google style guide.

Reply:
Why?

Reply:
It was not on purpose!